ceph-client.rb

```diff
@@ -52,9 +53,9 @@ def install
       -DWITH_BABELTRACE=OFF
       -DWITH_BLUESTORE=OFF
       -DWITH_CCACHE=OFF
-      -DWITH_CEPHFS=OFF
+      -DWITH_CEPHFS=ON
```

Comment:
i am not quite sure about this. i think what we need is but the client side of cephfs. while `WITH_CEPHFS` is supposed to enable the server side of cephfs. lemme try this on my build host to see if i can just enable `WITH_LIBCEPHFS` for building the client side. if i cannot, that'd be a bug.

Reply:
tested. please drop this change, adding "cephfs" to `targets` would suffice.

Reply:
done